Make 'indentation prefix' example easier to read (#12172)
lampepfl/dotty: docs/docs/reference/other-new-features/indentation.md

### Spaces vs Tabs

> Indentation prefixes can consist of spaces and/or tabs. Indentation widths are the indentation prefixes themselves, ordered by the string prefix relation. So for instance "2 tabs, followed by 4 spaces" is strictly less than "2 tabs, followed by 5 spaces", but "2 tabs, followed by 4 spaces" is incomparable to "6 tabs" since neither is the string prefix of the other. It is an error if the indentation width of some line is incomparable with the indentation width of the region that's current at that point. To avoid such errors, it is a good idea not to mix spaces and tabs in the same source file.

Reviewer: Here we loose another example. Maybe you can add it again after the end of this sentence, like: 
```suggestion
Indentation prefixes can consist of spaces and/or tabs. Indentation widths are the indentation prefixes themselves, ordered by the string prefix relation. So for instance "2 tabs, followed by 4 spaces" is strictly less than "2 tabs, followed by 5 spaces". However, "2 tabs, followed by 4 spaces" is both incomparable to "6 tabs" and incomparable to "4 spaces, followed by 2 tabs" -- neither is the string prefix of the other. It is an error if the indentation width of some line is incomparable with the indentation width of the region that's current at that point. To avoid such errors, it is a good idea not to mix spaces and tabs in the same source file.
```
